lib/OpenQA/Shared/Controller/Running.pm

```diff
@@ -113,10 +109,7 @@ sub streamtext ($self, $file_name, $start_hook = undef, $close_hook = undef) {
     # Setup utility function to close the connection if something goes wrong
     my $timer_id;
     my $close_connection = sub ($self) {
-        Mojo::IOLoop->remove($timer_id);
```

Comment:
Why are you removing the timer? I see no new code covering this here, and the timer is still being used 🤔

Reply:
`close_connection` calls finish which is taking care of this. that was exactly the objective.. to handle it consistently and not call it twice